### ISSUE TITLE
Interaction command

### ISSUE THREAD

**Issue by OndraM** [lmc-eu/matej-client-php#22]
- `Lmc\Matej\Model\Command\Interaction`
- Static named constructors for each interaction type (as in ItemPropertySetup command)
- We may abstract the public API from "e-shop" naming of the interactions and provide some domain-independent names of the static constructors.
  - Eg. `Interaction::view(...)` will be Matej's `detailviews` interaction
  - Similar for other types -  for `purchases` we may have `::conversion` (or something like that), also for `bookmarks` and `ratings` (I'm not sure about the abstract names though)
- `value` will be optional and `1.0` by default
- `context` will be also optional and `default` by default
- not sure if timestamp should be unix-timestamp or we should accept DateTimeInterface instead

Once the command is done, in separate commit (or in separate pull request) the EventsRequestBuilder should be extended by `addInteraction` and `addInteractions` methods.
